docs/DeveloperGuide.md

```diff
@@ -732,3 +732,10 @@ Timestamp in the status bar is updated.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+<br>
+
+2. Currently, when the `delete` command encounters invalid indexes, it generates an error and does not delete any patient or specialist records. 
+In contrast, the `delsc` command handles invalid shortcuts by recognizing and ignoring them, while successfully removing any arguments associated with valid shortcuts. 
```

Comment:
I think writing "arguments associated with valid shortcuts" makes the sentence a bit hard to understand. I think the word "argument" adds to confusion.
My suggestion for an alternative would be "while continuing to remove any valid shortcuts in the command."

Reply:
Noted with thanks. I have made the changes you suggested.